scripts/tutorial-patches/build.sbt.patch

```diff
@@ -5,7 +5,7 @@ index a633066..3df8b74 100644
 @@ -124,7 +124,7 @@ lazy val testchipip = (project in file("generators/testchipip"))
  
  lazy val chipyard = conditionalDependsOn(project in file("generators/chipyard"))
-   .dependsOn(boom, hwacha, sifive_blocks, sifive_cache, utilities,
+   .dependsOn(boom, hwacha, sifive_blocks, sifive_cache, utilities, iocell,
 -    sha3, // On separate line to allow for cleaner tutorial-setup patches
 +//    sha3, // On separate line to allow for cleaner tutorial-setup patches
```

Comment:
This should not be commented out?

Reply:
I think you're misreading the syntax highlighting- this is the patch. The only thing that's changing is the `iocells` part.

Reply:
Oh yeah, you are right.